README.md

```diff
@@ -20,14 +20,15 @@ Project specific settings can also be set for LSP-pyright (and all LSP plugins):
 
 - From the command palette `Project: Edit Project`
 
-Your project will also require a `pyrightconfig.json` file, usually in the project root.
-
-At a minimum, the file should define where your Python virtualenvs are located and the name of the one to use for your project:
+You may add a `pyrightconfig.json` file to your project root.
+At a minimum, the file should define where your Python `virtualenvs` are located
```

Comment:
That sentence is also not entirely true. One can have a configuration file without any venv-specific settings. I feel that the Virtual Environment case should have its own section and then this sentence would be relevant there.

Reply:
I guess I would remove this part of changes in this PR and let other people who are familiar with to write that section.